### PR TITLE
feat(apm): add language stubs

### PR DESCRIPTION
**Pull request by devfreddy**

### CODE REVIEW

File: recipes/newrelic/apm/java/java-install.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+name: java-agent-installer
+displayName: Java Agent Installer
+description: New Relic install recipe for instrumenting Java applications
+repository: https://github.com/newrelic/newrelic-java-agent
+
+installTargets:
+  - type: application
+    os: linux
+  - type: application
```
Comment (Julien4218) [inline recipes/newrelic/apm/java/java-install.yml:12]: I think rishi said we would focus on java/linux only for now. If we do need windows, it's most likely going to be a different recipe file (powershell versus linux/bash)

File: recipes/newrelic/apm/go/go-install.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+name: go-agent-installer
+displayName: Go Agent Installer
+description: New Relic install recipe for instrumenting Go applications
+repository: https://github.com/newrelic/go-agent
+
+installTargets:
+  - type: application
+    os: linux
+  - type: application
```
Comment (Julien4218) [inline recipes/newrelic/apm/go/go-install.yml:12]: Same comment below for java/windows

File: recipes/newrelic/apm/node/node-install.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+name: node-agent-installer
+displayName: Node Agent Installer
+description: New Relic install recipe for instrumenting Node applications
+repository: https://github.com/newrelic/node-newrelic
+
+installTargets:
+  - type: application
+    os: linux
+  - type: application
```
Comment (Julien4218) [inline recipes/newrelic/apm/node/node-install.yml:12]: Same comment than java windows

File: recipes/newrelic/apm/php/php-install.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+name: php-agent-installer
+displayName: PHP Agent Installer
+description: New Relic install recipe for instrumenting PHP applications
+repository: https://github.com/newrelic/newrelic-php-agent
+
+installTargets:
+  - type: application
+    os: linux
+  - type: application
```
Comment (Julien4218) [inline recipes/newrelic/apm/php/php-install.yml:12]: I dont think there is a PHP on windows? I know we dont have an agent for it

File: recipes/newrelic/apm/python/python-install.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+name: python-agent-installer
+displayName: Python Agent Installer
+description: New Relic install recipe for instrumenting Python applications
+repository: https://github.com/newrelic/newrelic-python-agent
+
+installTargets:
+  - type: application
+    os: linux
+  - type: application
```
Comment (Julien4218) [inline recipes/newrelic/apm/python/python-install.yml:12]: Same than java windows

File: recipes/newrelic/apm/ruby/ruby-install.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+name: ruby-agent-installer
+displayName: Ruby Agent Installer
+description: New Relic install recipe for instrumenting Ruby applications
+repository: https://github.com/newrelic/newrelic-ruby-agent
+
+installTargets:
+  - type: application
+    os: linux
+  - type: application
```
Comment (Julien4218) [inline recipes/newrelic/apm/ruby/ruby-install.yml:12]: same than java windows

**Review by Julien4218 — CHANGES_REQUESTED** [review]
I think the windows/language combo should go in their own file, and perhaps we ll add those when we'll do them?

File: recipes/newrelic/apm/java/java-install.yml (hunk 1 above)
Comment (devfreddy) [reply recipes/newrelic/apm/java/java-install.yml]: Split them out

File: recipes/newrelic/apm/go/go-install.yml (hunk 2 above)
Comment (devfreddy) [reply recipes/newrelic/apm/go/go-install.yml]: Split them out

File: recipes/newrelic/apm/node/node-install.yml (hunk 3 above)
Comment (devfreddy) [reply recipes/newrelic/apm/node/node-install.yml]: Split them out

File: recipes/newrelic/apm/php/php-install.yml (hunk 4 above)
Comment (devfreddy) [reply recipes/newrelic/apm/php/php-install.yml]: You can certainly run PHP on Windows, however, I did look it up, and you're correct, we don't have a PHP Agent on Windows!

Removing recipe for this.

File: recipes/newrelic/apm/python/python-install.yml (hunk 5 above)
Comment (devfreddy) [reply recipes/newrelic/apm/python/python-install.yml]: Split these up

File: recipes/newrelic/apm/ruby/ruby-install.yml (hunk 6 above)
Comment (devfreddy) [reply recipes/newrelic/apm/ruby/ruby-install.yml]: Split these up

**Review by Julien4218 — APPROVED** [review]

**Pull request merged**